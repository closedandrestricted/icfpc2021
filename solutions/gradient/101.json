{"vertices":[[21,76],[40,63],[53,37],[35,81],[22,55],[80,141],[89,147],[89,37],[68,57],[45,51],[54,77],[90,137],[78,106],[84,124],[60,80],[97,51],[70,76],[81,166],[81,65],[77,55],[54,64],[75,82],[58,78],[87,74],[82,46],[82,138],[84,80],[62,74],[84,152],[65,79],[66,83],[79,88],[79,92],[74,86],[62,81],[83,123],[71,92],[80,81],[68,97],[98,159],[80,103],[63,94],[82,74],[61,100],[82,109],[89,118],[76,114],[65,101],[69,128],[93,137],[85,125],[112,168],[94,105],[76,117],[119,143],[102,142],[52,146],[71,125],[60,135],[109,150],[125,155],[61,119],[70,111],[49,146],[30,156]]}