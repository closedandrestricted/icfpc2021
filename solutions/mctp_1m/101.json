{"vertices":[[39,68],[59,73],[38,57],[27,59],[50,51],[18,80],[29,87],[18,75],[44,67],[34,77],[29,78],[21,81],[46,67],[33,74],[40,75],[31,76],[45,86],[30,67],[31,73],[39,67],[39,90],[39,89],[45,97],[40,102],[29,63],[21,80],[42,96],[42,103],[17,67],[38,100],[35,102],[43,104],[43,100],[43,98],[54,94],[32,72],[45,104],[42,110],[40,106],[29,64],[42,109],[46,106],[39,102],[44,112],[48,110],[52,96],[50,117],[39,104],[39,105],[37,88],[43,94],[19,73],[38,103],[54,97],[35,83],[35,97],[35,85],[38,81],[35,68],[35,85],[36,70],[46,84],[35,77],[20,64],[40,63]]}