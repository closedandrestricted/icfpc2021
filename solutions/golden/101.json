{"vertices":[[21,78],[39,63],[52,37],[36,82],[21,57],[90,137],[81,146],[89,37],[67,56],[46,52],[58,78],[89,138],[78,106],[84,124],[70,65],[98,51],[75,74],[83,166],[80,65],[77,55],[54,66],[75,82],[60,79],[87,74],[83,46],[86,138],[84,80],[64,74],[86,152],[67,79],[66,83],[79,88],[79,92],[74,87],[62,82],[79,126],[69,92],[81,81],[68,97],[99,160],[81,103],[63,94],[83,88],[61,100],[82,109],[91,118],[75,114],[66,101],[68,128],[93,137],[84,123],[114,167],[94,105],[74,117],[119,143],[102,142],[50,146],[70,125],[60,136],[112,147],[127,154],[61,119],[68,109],[49,147],[30,157]]}